src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -83,9 +83,12 @@ public void InitializeService(ServiceHost serviceHost)
                 if (sqlConn != null)
                 {
                     DisasterRecoveryService.Instance.InitializeBackup(helper.DataContainer, sqlConn);
-                    BackupConfigInfo backupConfigInfo = DisasterRecoveryService.Instance.GetBackupConfigInfo(sqlConn.Database);
-                    backupConfigInfo.DatabaseInfo = AdminService.GetDatabaseInfo(connInfo);
-                    response.BackupConfigInfo = backupConfigInfo;                
+                    if (!connInfo.IsSqlDW)
```

Comment:
I don't get the same exception for azure. Kate is going to work on this area 

---
In reply to: [122093127](https://github.com/Microsoft/sqltoolsservice/pull/380#discussion_r122093127) [](ancestors = 122093127)